docs/continous_integration_report.md

```diff
@@ -0,0 +1,31 @@
+# Continuous Integration Report
+## Description
+We introduced GitHub Actions in our GitHub repository in order to have a continuous integration tool. Via this tool, we analyze whether the backend and the frontend of our application meets the following criteria:
+- builds successfully 
+- passes the written tests
+- passes a quality gate defined in Sonarcloud.
+
+### GitHub Actions specifications
+- The CI runs each time a new *pull request* from a *feature branch* to the *development branch* is created. 

```

Comment:
A CI minden PR esetén lefut, tehát ha dev->mastert csinálunk akkor is, ezért kérlek ezt írd át picit, nehogy belekossenek.

Reply:
igaz okés!